packages/account/src/providers/transaction-request/blob-transaction-request.ts

```diff
@@ -17,10 +18,7 @@ export interface BlobTransactionRequestLike extends BaseTransactionRequestLike {
 
 export class BlobTransactionRequest extends BaseTransactionRequest {
   static from(obj: BlobTransactionRequestLike) {
-    if (obj instanceof this) {
-      return obj;
-    }
-    return new this(obj);
+    return new this(clone(obj));
```

Comment:
My preference would be to use the native [`structuredClone` algorithm](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) but this was not returning an [exact copy](https://github.com/FuelLabs/fuels-ts/actions/runs/10689518036/job/29631794565?pr=3097) , I'm not too sure why though